README.md

```diff
@@ -1,55 +1,55 @@
 # Dash Bio
 
-[![CircleCI](https://circleci.com/gh/plotly/dash-bio/tree/master.svg?style=svg)](https://circleci.com/gh/plotly/dash-bio)
+Dash Bio is a suite of bioinformatics components built to work with
+[Dash](https://github.com/plotly/dash/).
 
-🚧 *Work-in-Progress* 🏗️
+Demo:
+[https://dash-bio.plotly.host/Portal/](https://dash-bio.plotly.host/Portal/)
 
-Open-source bioinformatics components for Dash
+Documentation:
+[https://dash.plot.ly/dash-bio](https://dash.plot.ly/dash-bio)
 
-Demo: https://dash-bio.plotly.host/Portal/
+## Components
 
-## Progress
+The Dash Bio components each fall into one of three categories:
 
-#### Bioinformatics charts
+- Custom chart types
+- Sequence analysis tools
+- Visualization tools
```

Comment:
The third category is extremely general and includes the first two categories, so I don't think that division makes a lot of sense... Where does it come from?